src/ResourceManager/TrafficManager/Commands.TrafficManager2.Test/App.config

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <runtime>
+    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
+      <dependentAssembly>
+        <assemblyIdentity name="Newtonsoft.Json" publicKeyToken="30ad4fe6b2a6aeed" culture="neutral" />
+        <bindingRedirect oldVersion="0.0.0.0-9.0.0.0" newVersion="9.0.0.0" />
```

Comment:
Reviewers: is this assembly binding also needed in the the cmdlet project? Visual Studio added it when upgrading packages but I see that app.config files are ignored by the azure-powershell git repo. Can you advise, please? Thank you